### PR TITLE
Add gRPC Example for NI-SCOPE

### PR DESCRIPTION
**Pull request by IPetersNI**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

- [ ] I've added tests applicable for this pull request
N/A
### What does this Pull Request accomplish?

Add gRPC Examples for NI-SCOPE. These drivers have their protos and system tests added and so we are adding an example for each to show the use of the gRPC sessions. These examples are based off of existing examples so that users can see side-by-side the difference in setting up session (e.g. how easy it is).

### What testing has been done?
Ran local server and after installing build artifacts successfully ran the gRPC examples.

### CODE REVIEW

**Comment by marcoskirsch** [comment]
Feedback from #1852 applies here as well, please look at that and apply it here.

**Comment by IPetersNI** [comment]
Removed the todo and we have a work item to cover adding the tests back : https://dev.azure.com/ni/DevCentral/_workitems/edit/2222417

**Comment by marcoskirsch** [comment]
please fix merge conflicts

**Review by dmondrik — APPROVED** [review]
LGTM

File: src/niscope/examples/niscope_fetch_grpc.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import grpc
+import niscope
+import pprint
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+def example(resource_name, channels, options, length, voltage, address, port,):
+    session_name = ''  # user-specified name; empty string means use a new , unnamed session
```
Comment (ni-jfitzger) [inline src/niscope/examples/niscope_fetch_grpc.py:13]:
```suggestion
session_name = '' # user-specified name; empty string means use a new, unnamed session
```
grammar

File: src/niscope/examples/niscope_fetch_grpc.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import argparse
+import grpc
+import niscope
+import pprint
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+def example(resource_name, channels, options, length, voltage, address, port,):
```
Comment (ni-jfitzger) [inline src/niscope/examples/niscope_fetch_grpc.py:12]:
```suggestion
def example(resource_name, channels, options, length, voltage, address, port):
```
The style inconsistency of this extra comma kind of bothers me.

File: src/niscope/examples/niscope_fetch_grpc.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import grpc
+import niscope
+import pprint
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+def example(resource_name, channels, options, length, voltage, address, port):
+    session_name = '' # user-specified name; empty string means use a new, unnamed session
```
Comment (ni-jfitzger) [inline src/niscope/examples/niscope_fetch_grpc.py:13]: Sorry, my initial comment was a bit off with some of the spacing. I updated it, but it looks like you committed the original. You may need to readd that second space before the comment to pass the flake8 check. If not, it's fine as is.

Comment (ni-jfitzger) [reply src/niscope/examples/niscope_fetch_grpc.py:13]: Yeah, you'll need to readd that second space.

**Review by ni-jfitzger — APPROVED** [review]

**Pull request merged**